webapp/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:16 AS build
+FROM node:24 AS build
```

Comment:
There are other places we would want to bump the Node.js version including...

https://github.com/mcagov/droits/blob/main/webapp/package.json#L5

https://github.com/mcagov/droits/blob/main/.tool-versions#L2

Note that the latter would also affect `backoffice`, so we would maybe override that in a `webapp/.tool-versions` file.